editor/src/components/canvas/right-menu.tsx

```diff
@@ -69,8 +69,8 @@ export const RightMenuTile: React.FunctionComponent<RightMenuTileProps> = (props
     <SimpleTile
       style={{
         ...props.style,
-        width: UtopiaTheme.layout.rowHeight.mediumLarge,
-        height: UtopiaTheme.layout.rowHeight.mediumLarge,
+        width: 38,
```

Comment:
How come these (and later values too) weren't updated in the `UtopiaTheme`? Should they be?

Reply:
Ok looks like they were actually removed from there. Please can you put them re-add them in a sensible place.

Reply:
There weren't many places of components using deleted values such as UtopiaTheme.lay...mediumLarge, and we considered them to be unusual values so instead, we just set them in style to indicate that. Do you still think I should re-add them in?

Reply:
it sounds like if those are rarely used and unusual, it's better to have them locally 👍

Reply:
Ok sure that's fine then